package.json

```diff
@@ -52,6 +52,6 @@
   },
   "dependencies": {
     "bootstrap": "^5.3.1",
-    "pdfjs-dist": "^2.14.305"
+    "pdfjs-dist": "^2.16.105"
```

Comment:
Let's keep the older version, since it doesn't fix anything right now, and it means there are no whitespace changes in the extracted text.